airbyte-integrations/connectors-performance/destination-harness/src/main/java/io/airbyte/integrations/destination_performance/PerformanceHarness.java

```diff
@@ -165,6 +169,16 @@ void runTest() throws Exception {
           log.info("current throughput({}): {} total MB {}", counter, (totalBytes / MEGABYTE) / ((System.currentTimeMillis() - start) / 1000.0),
               totalBytes / MEGABYTE);
         }
+
+        // If sync mode is incremental, send state message every 10,000 records
+        if (syncMode.equals("INCREMENTAL") && counter % STATE_FREQUENCY == 0) {
+          final AirbyteMessage stateMessage = new AirbyteMessage()
+              .withType(Type.STATE)
+              .withState(new AirbyteStateMessage()
+                  .withType(AirbyteStateMessage.AirbyteStateType.GLOBAL)
+                  .withData(Jsons.deserialize("{\"checkpoint\": \"" + counter + "\"}")));
+          destination.accept(stateMessage);
```

Comment:
Just for me to understanding: what does destination do with a state in this case?